DESCRIPTION

```diff
@@ -43,12 +43,15 @@ Imports:
     rlang,
     smoothqr,
     stats,
+    stringr,
+    stringi,
     tibble,
     tidyr,
     tidyselect,
     usethis,
     vctrs,
-    workflows (>= 1.0.0)
+    workflows (>= 1.0.0),
+    zoo
```

Comment:
`zoo` I needed to remove the potential extra `NA`'s that I introduced by the extra shifting. Not sure whether we want to leave that to `step_epi_naomit`. If there are in fact `NA`'s at the end in the original dataset, this will also trim those out.

Reply:
Let's just defer `NA` removal to the explicit step function.